docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,8 @@
 ### What was the problem?
+This PR resolves #INSERT_ISSUE_NUMBER
 
 ### How did I solve it?
+<!--- Describe your technical implementation -->
 
-### How to manually test it?
-
-### Review checklist
-
-- [ ] The PR resolves #INSERT_ISSUE_NUMBER
-- [ ] All new code is covered with unit tests
-- [ ] Relevant integration / functional tests are added
-- [ ] Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
-- [ ] Documentation has been added/updated
+### How this has been tested?
+<!--- Please describe how you tested your changes -->
```

Comment:
This should be consistent with the above phrase

Reply:
Thanks for your feedback. Updated based on your comments

Reply:
I think the proper form is `How was it tested?`